GameData/RealismOverhaul/RO_DependentMods/RO_AJE.cfg

```diff
@@ -37,6 +37,201 @@
 	%title = Avon RB-146 Mk.302 Turbojet
 }
 
+// Clone aje_j57 to be J48
++PART[aje_j75]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@name = RO-J48
+	@MODEL,0
+	{
+		%scale = 0.6217, 0.6217, 0.6217
+	}
+
+	!MODEL,1 {}
+
+	// ven's 1.x used these paths...
+	MODEL:NEEDS[VenStockRevamp/Squad]
+	{
+		model = VenStockRevamp/Squad/Parts/Propulsion/EngineCore-Medium
+		position = 0.0, -0.8, 0.0
+		scale = 0.955,1.55,0.955
+	}
+	MODEL:NEEDS[VenStockRevamp/Squad]
+	{
+		model = VenStockRevamp/Squad/Parts/Propulsion/EngineCore-Small
+		position = 0.0, 0.45, 0.0
+		scale = 1.56,0.6,1.56
+	}
+	MODEL:NEEDS[VenStockRevamp/Squad]
+	{
+		model = VenStockRevamp/Squad/Parts/Propulsion/EngineCore-Small
+		position = 0.0, 0.45, 0.0
+		rotation = 180, 0, 0
+		scale = 1.56,0.03,1.56
+	}
+
+	// phineas' ven's 2.x uses these paths. 2.x is needed for restock
+	// compatibility in ksp 1.6. We add models for both paths;
+	// the bad path will just get ignored
+	MODEL:NEEDS[VenStockRevamp]
+	{
+		model = VenStockRevamp/Assets/Engine/EngineCore-Medium
+		position = 0.0, -0.8, 0.0
+		scale = 0.955,1.55,0.955
+	}
+	MODEL:NEEDS[VenStockRevamp]
+	{
+		model = VenStockRevamp/Assets/Engine/EngineCore-Small
+		position = 0.0, 0.45, 0.0
+		scale = 1.56,0.6,1.56
+	}
+	MODEL:NEEDS[VenStockRevamp]
+	{
+		model = VenStockRevamp/Assets/Engine/EngineCore-Small
+		position = 0.0, 0.45, 0.0
+		rotation = 180, 0, 0
+		scale = 1.56,0.03,1.56
+	}
```

Comment:
Oof.  Since RO will not release for anything less than KSP 1.8 and the current version of VSR... please check VSR's current master branch and only use the correct path from there.

Reply:
I just copied what already existed without looking too closely. I'll clean it up.